meson.build

```
@@ -0,0 +1,56 @@
+project(
+    'python-rtmidi',
+    'cpp',
+    version: '1.5.0rc1',
+    license: 'MIT',
+    default_options: [
+        'warning_level=2'
+    ],
+    meson_version: '>=0.53.0'
+)
+
+# Dependencies
+cpp = meson.get_compiler('cpp')
+jack2_dep = dependency('jack', version:'>=1.9.11', required: false)
+jack1_dep = dependency('jack', version:'>=0.125.0, <1.0', required: false)
```

```suggestion
jack1_dep = dependency('jack', version:['>=0.125.0', '<1.0'], required: false)
```

Currently this incorrectly compares 1.9.x versions as valid jack1 found dependencies.

---

Fixed in 53b6921.